pkg/policies/branch/branch.go

```diff
@@ -88,23 +97,35 @@ type RepoConfig struct {
 
 	// BlockForce overrides the same setting in org-level, only if present.
 	BlockForce *bool `yaml:"blockForce"`
+
+	// RequireUpToDateBranch : set to true to require that branches must be up
+	// to date before merging. Default false.
+	RequireUpToDateBranch *bool `yaml:"requireUpToDateBranch"`
+
+	// RequireStatusChecks is a list of status checks (by name) that are required in
+	// order to merge into the protected branch.
+	RequireStatusChecks []string `yaml:"statusChecks"`
```

Comment:
Update the comment to indicate how the repo-config is merged with the org-config. Ex:
```
RequireStatusChecks adds more check names to the org-level list. Does not
override. Always allowed irrespective of DisableRepoOverride setting.
```
Secondly, is this the behavior that we want? Is there a situation where a repo might want to remove/replace the org-level list?

Reply:
Fixed in 754144bd3f9e24f3554793450990d44d4116b98f

> Secondly, is this the behavior that we want? Is there a situation where a repo might want to remove/replace the org-level list?

This was a good point, and after thinking about it, I switched to a replacement instead of merging. If we do merging, then there is no way for the repo override to remove checks that may not be relevant to the repo (e.g., a go lint check on a nodejs project). At least with replacement, you can simulate a merge by repeating the list from the org.